doc/user_manual/postprocessor.tex

```diff
@@ -306,7 +306,7 @@ \subsubsection{ComparisonStatistics}
 in each bin, it creates a cdf function numerically, and from the cdf
 takes the derivative to generate a pdf.  It also calculates statistics
 of the data such as mean and standard deviation. The post-processor
-can generate either a CSV file or a PointSet.
+can generate a CSV file only.
```

Comment:
just for the record: what was the rationale behind it?

Reply:
because the PointSet output was not standard